testing/colormake/APKBUILD

```diff
@@ -1,17 +1,18 @@
 # Contributor: Henrik Riomar <henrik.riomar@gmail.com>
 # Maintainer: Henrik Riomar <henrik.riomar@gmail.com>
 pkgname=colormake
-pkgver=0.9.20140503
+pkgver=0.9.20170221
 pkgrel=0
+_sha=93dd19b5143b124208caab5e9b6d57b5e6ec02cb
```

Comment:
I want avoid adding random git snapshots if possible. Could you ask if upstream could make a new tag?

Reply:
@ncopa sure done https://github.com/pagekite/Colormake/issues/22

Reply:
@ncopa upstream does not seem to be active on github at the moment: https://github.com/pagekite

93dd19b5143b124208caab5e9b6d57b5e6ec02cb is the latest in HEAD (not moved since 20170221), do you think we can merge this PR while waiting for upstream? I am watching the upstream repo for changes, so if it changes I can prepare a new PR.